tempodb/encoding/vparquet4/block_traceql.go

```diff
@@ -898,12 +941,14 @@ const (
 	otherEntrySpanKey    = "span"
 	otherEntryEventKey   = "event"
 	otherEntryLinkKey    = "link"
+	otherEntryScopeKey   = "scope"
 
 	// a fake intrinsic scope at the trace lvl
 	intrinsicScopeTrace = -1
 	intrinsicScopeSpan  = -2
 	intrinsicScopeEvent = -3
 	intrinsicScopeLink  = -4
+	intrinsicScopeScope = -5
```

Comment:
PR is looking good. Do we want to clean up the remaining references to scope scope?  Looks like there are a handful.